vladiate/vlad.py

```diff
@@ -88,31 +93,39 @@ def _log_missing(self, missing_items):
 
     def validate(self):
         self.logger.info(
-            "\nValidating {}(source={})".format(self.__class__.__name__, self.source)
+            "Validating {}(source={})".format(self.__class__.__name__, self.source)
```

Comment:
Any reason we're removing the newline here?

Reply:
Remove this newline o avoid an extra blank line in the log file.